scripts/close_boundary.py

```diff
@@ -1,36 +1,59 @@
 #!/usr/bin/env python3
 
 from pycascadia.loaders import load_source
+from pygmt import grdcut
+import matplotlib.pyplot as plt
 import argparse
 
-def replace_land_with_zeros(arr, test_arr=None):
+def replace_land_with_zeros(arr, test_arr=None, value=0.0):
```

Comment:
Maybe worth renaming due to functionality added?
something like
```suggestion
def clip_to_value(arr, test_arr=None, value=0.0):
```
(not sure this is the best name?)